test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
I tried several ways, but this is only way to work I think. Yes, let me add some comments about versions.